CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# **Gearman welcomes contributions from the open-source community!**
+
+### Please follow the steps given below to make your first contribution ->
+You can obtain the latest source tree for Gearman by issuing the following command:
+
+`git clone https://github.com/gearman/gearmand`
+
+You will need to have git installed in order for this to work.
+
+If you are building from source control (i.e. git), you should setup your environment via: \
+`./bootstrap.sh autoreconf`
+
+This will build the files you will need in order to run `./configure`
+
+If you are working on the server, here are some handy environmental variables you can set so that you can debug make test:
+
+ * GEARMAN_VALGRIND <-- runs the server under valgrind.
+ * GEARMAN_MANUAL_GDB <-- runs the server under a remote version of GDB.
+ * GEARMAN_LOG <-- generates a log file for you with the output of the debug for the server
+
+GDB will pause the server while you do run the GDB command.
```

Comment:
Please add "Note that " to the beginning of this line and change "do run" to just "run".